src/main/java/Ultron/Tasks/Event.java

```diff
@@ -0,0 +1,142 @@
+package ultron.tasks;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class Event extends Task {
+    /** Stores the regex for parsing the command for event. */
+    private static final Pattern DATE_MATCH =
+        Pattern.compile("^(.*) (/at) (.*)$");
+    /** Stores the date format regex.*/
+    private static final DateFormat DATE_FORMAT =
+        new SimpleDateFormat("dd-MM-yyyy HHmm");
```

Comment:
Would it be simpler if you use LocalDateTime here?

Reply:
Yes, that might be true. Might edit in a future commit haha. Thanks for letting me know.